docs/content/docs/ops/state/unaligned_checkpoints.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Unaligned checkpoints"
+weight: 9
+type: docs
+aliases:
+- /ops/state/unalgined_checkpoints.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+# Unaligned checkpoints
+
+Starting with Flink 1.11, checkpoints can be unaligned.
+[Unaligned checkpoints]({{< ref "docs/concepts/stateful-stream-processing" >}}#unaligned-checkpointing) 
+contain in-flight data (i.e., data stored in buffers) as part of the checkpoint state, allowing
+checkpoint barriers to overtake these buffers. Thus, the checkpoint duration becomes independent of
+the current throughput as checkpoint barriers are effectively not embedded into the stream of data
+anymore.
+
+You should use unaligned checkpoints if your checkpointing durations are very high due to
+backpressure. Then, checkpointing time becomes mostly independent of the end-to-end latency. Be
+aware unaligned checkpointing adds to I/O to the state backends, so you shouldn't use it when the
```

Comment:
replace:
> aware unaligned checkpointing adds to I/O to the state backends, so you shouldn't use it when the I/O to the state backend is actually the bottleneck during checkpointing.

with:
> aware unaligned checkpointing adds additional I/O overhead, so you shouldn't use it when the I/O is actually the bottleneck during checkpointing.

? ("state backends" have nothing to do with uc)

Reply:
We now use `checkpoint storage` to refer to the system hosting the checkpoints.

Reply:
I'll change to `checkpoint storage`

fyi: I copied this over from the current docs. I moved the UC section to a separate page.

Reply:
Yeah, I know that I'm nit picking to technically a wrong person ;) But I thought that while you were already touching this and I noticed it, then let's fix it anyway.

Reply:
Sure, just letting you know.